src/components/ErrorPage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './../styles/ErrorPage.css'
+import Navbar from './Navbar'
+import './../styles/App.css'
+
+
+function ErrorPage() {
+    return (
+        <>    <Navbar /> 
```

Comment:
AuthRoute already contains the Navbar, so this wasn't needed. Reusing components is always preferable. That's why I suggested AuthRoute.